packages/typos-lsp/package.yaml

```
@@ -20,12 +20,18 @@ source:
     - target: linux_x64_gnu
       file: typos-lsp-{{ version }}-x86_64-unknown-linux-gnu.tar.gz
       bin: typos-lsp
+    - target: linux_x64_musl
```

The `_musl` target is only necessary when the binary is dynamically linked, which is rarely the case for musl binaries.

```suggestion
    - target: linux_x64
```